### PR TITLE
Docker facelift

### PR DESCRIPTION
**Pull request by kirkkwang**
bringing changes from BL's `docker_facelift`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG HYRAX_IMAGE_VERSION=3.0.3
```
Comment (orangewolf) [inline Dockerfile:1]: why isn't this `ARG HYRAX_IMAGE_VERSION=v4.0.0.beta2` to match BL?

**Review by orangewolf — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,55 +1,100 @@
...
+  sed -i '/require .enumerator./d' /usr/local/bundle/gems/edtf-3.0.5/lib/edtf.rb"
 COPY --chown=1001:101 $APP_PATH/bin/db-migrate-seed.sh /app/samvera/
-
 COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 
-ARG SETTINGS__BULKRAX__ENABLED="true"
-RUN mkdir -p /app/samvera/branding && ln -s /app/samvera/branding /app/samvera/hyrax-webapp/public/branding && \
-  yarn install && \
-  RAILS_ENV=production SECRET_KEY_BASE=`bin/rake secret` DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile
+ARG SETTING__BULKRAX__ENABLED="true"
```
Comment (laritakr) [inline Dockerfile:94]: Should this be SETTINGS__BULKRAX__ENABLED="true"?

File: Gemfile [hunk 3]
```diff
@@ -89,15 +89,12 @@ group :development do
...
-  gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "35970cee7b1e48d00c3fc838d2ca5790c09e2c9c"
-  # rubocop:enable Metrics/LineLength
-  gem 'willow_sword', git: 'https://github.com/notch8/willow_sword.git'
-end
+# Until b857a25becefc4bd960f25647033dfb2e899386e is in a major release, use this ref or a ref
+# who's ancestors include this ref.
+# rubocop:disable Metrics/LineLength
+gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "35970cee7b1e48d00c3fc838d2ca5790c09e2c9c"
+# rubocop:enable Metrics/LineLength
+gem 'willow_sword', git: 'https://github.com/notch8/willow_sword.git'
```
Comment (laritakr) [inline Gemfile:97]: Should willow_sword also be removed since it isn't used?

Comment (ShanaLMoore) [reply Gemfile:97]: Do you know which version of bulkrax no longer requires it?

Comment (kirkkwang) [reply Gemfile:97]: @ShanaLMoore looks like this [commit](https://github.com/samvera-labs/bulkrax/commit/ab33927e5eb7b80d8737215c9e3f0dbf0707249d) removed it, but i'm unsure how to check what version it is

Comment (ShanaLMoore) [reply Gemfile:97]: Thanks for digging that up. It should be OK then. They are on version 4.4.0 which should include that commit/change. Thanks for pointing it out @laritakr

Comment (ShanaLMoore) [reply Gemfile]: Ps. I found that out by looking through Bulkrax's commit history

File: docker-compose.yml [hunk 4]
```diff
@@ -3,14 +3,18 @@ version: '3.8'
 x-app: &app
   build:
     context: .
-    target: hyku-base
+    target: hyku-web
     args:
       - SETTINGS__BULKRAX__ENABLED=true
+    cache_from:
+      - ghcr.io/scientist-softserv/adventist-dl/base:latest
+      - ghcr.io/scientist-softserv/adventist-dl/web:latest
```
Comment (kirkkwang) [inline docker-compose.yml:11]: i'm rebuilding with the changes and I keep seeing this

image

could there be a naming issue?

Comment (laritakr) [reply docker-compose.yml:11]: Those are the same lines we tried adding to BL and removed again for the same reason. 

Does it build with them as is in spite of the errors? Wondering if we should remove them, or leave but note as something to be addressed.

Comment (kirkkwang) [reply docker-compose.yml:11]: @laritakr yup the app still spins up just fine

**Comment by jeremyf** [comment]
I spun this up locally and things are looking great.

**Pull request merged**